libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -153,3 +158,16 @@ export function expandTrigger({ element } = {}) {
 }
 
 export const yieldToMain = () => new Promise((resolve) => { setTimeout(resolve, 0); });
+
+export const lanaLog = ({ message, e = '' }) => window.lana.log(`${message} ${e.reason || e.error || e.message || e}`, {
+  clientId: 'feds-milo',
+  sampleRate: 1,
+});
+
+export const logErrorFor = async (fn, message) => {
```

Comment:
Do you think we should add this directly to lana.js?  Also wonder if we should have a sync and async version.

Reply:
Might be overkill, since usually lana consumers can rely on implicit logging.

All this wrapping just makes the code more complex, and we shouldn't encourage that. GNAV doesn't have much of a choice. Milo blocks should be able to rely on this https://github.com/adobecom/milo/blob/main/libs/utils/utils.js#L376